x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/token/CreateTokenRequest.java

```diff
@@ -138,6 +127,32 @@ public ActionRequestValidationException validate() {
         return validationException;
     }
 
+    private static ActionRequestValidationException missingFieldValidation(String field, String fieldValue,
```

Comment:
My preference is for these to be named based on what it _expected_, rather than what causes it to _fail_.

```suggestion
    private static ActionRequestValidationException validateRequiredField(String field, String fieldValue,
```